### PR TITLE
remove fake-apiserver to repo clusterpedia-io/fake-apiserver

### PR DESCRIPTION
**Pull request by liangyongzhenya**
remove fake-apiserver to repo clusterpedia-io/fake-apiserver

### CODE REVIEW

**Comment by calvin0327** [comment]
@liangyongzhenya We haven't used `go vendor` yet.  we can leave the go vendor directory uninitialized in the pr.

File: go.mod [hunk 1]
```diff
@@ -6,44 +6,9 @@ require (
...
-	k8s.io/kubernetes => github.com/k3s-io/kubernetes v1.24.0-k3s1
-	k8s.io/legacy-cloud-providers => github.com/k3s-io/kubernetes/staging/src/k8s.io/legacy-cloud-providers v1.24.0-k3s1
-	k8s.io/metrics => github.com/k3s-io/kubernetes/staging/src/k8s.io/metrics v1.24.0-k3s1
-	k8s.io/mount-utils => github.com/k3s-io/kubernetes/staging/src/k8s.io/mount-utils v1.24.0-k3s1
-	k8s.io/pod-security-admission => github.com/k3s-io/kubernetes/staging/src/k8s.io/pod-security-admission v1.24.0-k3s1
-	k8s.io/sample-apiserver => github.com/k3s-io/kubernetes/staging/src/k8s.io/sample-apiserver v1.24.0-k3s1
-	k8s.io/sample-cli-plugin => github.com/k3s-io/kubernetes/staging/src/k8s.io/sample-cli-plugin v1.24.0-k3s1
-	k8s.io/sample-controller => github.com/k3s-io/kubernetes/staging/src/k8s.io/sample-controller v1.24.0-k3s1
-)
+)
```
Comment (Iceber) [inline go.mod:14]: Need to fix this tip.

others lgtm

**Review by Iceber — APPROVED** [review]
LGTM

**Pull request merged**